<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="teacherTable">
      <property name="columnCount">
       <number>1</number>
      </property>
      <column>
       <property name="text">
        <string>Teacher Name</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="rulesTable">
      <property name="columnCount">
       <number>3</number>
      </property>
      <column>
       <property name="text">
        <string>Op</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>A</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>B</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="generateBtn">
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>

  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>

  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
